app/Http/Controllers/V2/Forms/UpdateFormController.php

```diff
@@ -38,6 +38,17 @@ public function __invoke(Form $form, UpdateFormRequest $updateFormRequest): Form
             }
         }
 
+        foreach (data_get($updateFormRequest, 'form_sections', []) as $formSection) {
+            $currentFormSection = $this->updateFormSection($formSection, $form);
+
+            foreach (data_get($formSection, 'form_questions', []) as $formQuestion) {
+                $this->handleQuestions($formQuestion, $currentFormSection);
+                if (data_get($formQuestion, "input_type") == "select") {
+                    $this->updateFormQuestion($formQuestion, $currentFormSection);
+                }
```

Comment:
There is a double for-each loop above doing the same thing, except for this conditional.

Wouldn't it be better to handle this on the above loop?